src/libcore/iter/adapters/mod.rs

```diff
@@ -1789,6 +1799,17 @@ impl<I> DoubleEndedIterator for Fuse<I> where I: DoubleEndedIterator {
         }
     }
 
+    #[inline]
+    default fn nth_back(&mut self, n: usize) -> Option<<I as Iterator>::Item> {
```

Comment:
Bikeshed: What does `default` keyword do here? I cannot find its meaning
in [Rust Reference book](https://doc.rust-lang.org/nightly/reference/keywords.html) and unstable book.

Reply:
`default` denotes that `nth_back` may be further specialized by a more specific trait implementation. Since specialization is unstable, it is not described in the reference.

Reply:
It looks like the original RFC is [2532](https://github.com/rust-lang/rfcs/blob/master/text/2532-associated-type-defaults.md)? It's interesting. Thanks for your help.

Btw, how can I mark this conversation as resolved.

Reply:
The original RFC is https://github.com/rust-lang/rfcs/pull/1210.